openpype/hosts/maya/plugins/publish/collect_review.py

```diff
@@ -97,7 +94,7 @@ def process(self, instance):
             data["frameStart"] = instance.data["frameStart"]
             data["frameEnd"] = instance.data["frameEnd"]
             data['step'] = instance.data['step']
-            data['fps'] = instance.data['fps']
+            data['fps'] = instance.context.data['fps']
```

Comment:
Context is already assigned to a variable higher up.
```suggestion
            data['fps'] = context.data['fps']
```